docs/overview/creating_effects.md

```diff
@@ -53,25 +47,27 @@ An `Option` can be converted into a ZIO effect using `ZIO.fromOption`:
 val zoption: IO[Option[Nothing], Int] = ZIO.fromOption(Some(2))
 ```
 
-The error type of the resulting effect is `Option[Nothing]`, signifying that the returned value is `None`. You can transform the `Option[Nothing]` into a more specific error type using `orElseFail`:
+The error type of the resulting effect is `Option[Nothing]`, signifying that if such an effect fails, with will fail with the value `None` (which has type `Option[Nothing]`).
```

Comment:
```suggestion
The error type of the resulting effect is `Option[Nothing]`, signifying that if such an effect fails, it will fail with the value `None` (which has type `Option[Nothing]`).
```